_includes/downloads-scala3.html

```diff
@@ -0,0 +1,136 @@
+  through <a href="https://get-coursier.io/docs/cli-installation">Coursier</a>.
+</p>
+
+<div class="download-options">
+  <div class="download-left">
+    <span class="or">or</span>
+    <a href="https://get-coursier.io/docs/cli-installation" class="btn-download">
+      <i class="fa fa-download"></i>
+      <span>Download Coursier</span>
+    </a>
+    <div class="text-variant">
+      <p>Best if you want to get <strong>started quickly</strong>.</p>
+      <p><i>Coursier will install all the necessary dependencies for you, so after
+         installing Coursier you are ready to go.</i></p>
+    </div>
+  </div>
+  <div class="download-right">
+    <a href="https://www.scala-sbt.org/download.html"" class="btn-download">
+      <i class="fa fa-download"></i>
+      <span>Download Sbt</span>
+    </a>
+    <div class="text-variant">
+      <p>Best if you want to work on <strong>larger projects</strong>.</p>
+      <p><i>If you already have sbt installed, it is very easy to switch to Scala version {{page.release_version}}.</i></p>
+    </div>
+  </div>
+</div>
+</div>
+
+<h2>Installing Scala 3 with <strong>Coursier</strong></h2>
+<div class="install-steps">
+  <div class="step arrow">
+    <img src="/resources/img/download/arrow-asset.png" alt="">
+    <div class="number-step">1</div>
+    <div class="text-step">
+      <h3>Install the <i>Coursier</i> Scala artifact fetcher.</h3>
+      <p>Visit the <a href="https://get-coursier.io/">Coursier website</a> for detailed installation instructions on how to install Coursier.</p>
+      <p><i>(Coursier is the simplest way to get started with Scala, since it will automatically download and install all necessary dependencies for you).</i></p>
+    </div>
+  </div>
+  <!-- <div class="step">
+    <div class="number-step">2</div>
+    <div class="text-step">
+      <h3>Then, prepare your environment</h3>
+      <p>Open a terminal and run the following command: <code>cs setup</code></p>
+      <p><i>(Running this <a href="https://get-coursier.io/docs/cli-setup">command</a> installs the correct Java version along with some other helpful command line tools).</i></p>
+    </div>
+  </div> -->
+  <div class="step arrow">
+    <img src="/resources/img/download/arrow-asset.png" alt="">
+    <div class="number-step">2</div>
+    <div class="text-step">
+      <h3>Then, install Scala 3</h3>
+      <p>Open a terminal and run the following commands:</p>
+      <p><code>cs install scala3-compiler</code><span>to install the Scala 3 compiler</span></p>
+      <p><code>cs install scala3-repl</code><span>to install the Scala 3 interactive read-eval-print loop</span></p>
+    </div>
+  </div>
+  <div class="step">
+    <div class="number-step">3</div>
+    <div class="text-step">
+      <h3>Use Scala 3</h3>
+      <p>To use the Scala 3 compiler to compile a file <code>test.scala</code> simply run<br><code>scala3-compiler test.scala</code> in your terminal.</p>
```

Comment:
I don’t think this is useful advice, this is very far from real-world usage where projects have multiple files and third-party library dependencies.
I don’t think it’s a good idea to teach this approach to the users, since they will have to de-learn it after a few hours and re-learn how to work on proper Scala projects.
I would start with a simple project seed, whose build definition is minimalist and not intimidating, but that scales to a “regular” project (ie, a project with more than one file, and possible some library dependencies).

Reply:
I just wanted to somehow mention that these commands now exist after installing them above. Otherwise it is near difficult to newcomers to guess the command names. What would you suggest?

Reply:
> otherwise it's not obvious how to actually get it to run.

Agreed!

Reply:
I slightly rephrased it to: 
![image](https://user-images.githubusercontent.com/408265/118652063-7c421a00-b7e6-11eb-8e53-1c707f21316a.png)

Is there a good resource to point people to after this paragraph? The getting started page is not more detailed on how to use the REPL.

Reply:
I think it might be best simply to say something like "If experimenting with the above leaves you interested for more, head on to the next section in which we explain what you need to work with Scala on larger projects".

Reply:
I was actually thinking about resources on the REPL. Of course sbt is an alternative for larger projects (and there is a lot of docs about it) but IIUC we are missing a basic getting started guide on how to use the REPL and experiment from there on.